.jenkins/caffe2/test.sh

```diff
@@ -53,7 +53,12 @@ cd ${INSTALL_PREFIX}
 
 # C++ tests
 echo "Running C++ tests.."
-for test in $INSTALL_PREFIX/test/*; do
+for test in $(find $INSTALL_PREFIX/test/); do
+  # Skip directories
+  if [ $(dirname "${test}") == 'aten' ]; then
+    "$test" -r=xml -o "${test}.xml"
```

Comment:
Could you double check in our job-dsl repo which directories does it collect xml report files?